### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by Rudiachenko**

### CODE REVIEW

File: src/main/java/com/dev/cinema/dao/impl/MovieDaoImpl.java [hunk 1]
```diff
@@ -1,25 +1,32 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
 
-@Dao
+@Repository
 public class MovieDaoImpl implements MovieDao {
     private static final Logger logger = Logger.getLogger(MovieDaoImpl.class);
+    private final SessionFactory sessionFactory;
+
+    @Autowired
```
Comment (DmGrt) [inline src/main/java/com/dev/cinema/dao/impl/MovieDaoImpl.java:19]: I think we don't need this annotation here. SAme with other classes

**Review by DmGrt — COMMENTED** [review]
Good job!

File: pom.xml [hunk 2]
```diff
@@ -36,11 +38,70 @@
...
+
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-dbcp2</artifactId>
+            <version>2.1.1</version>
+        </dependency>
+
+        <dependency>
+            <groupId>javax.servlet.jsp.jstl</groupId>
+            <artifactId>javax.servlet.jsp.jstl-api</artifactId>
```
Comment (Sofasmile) [inline pom.xml]: do we need this dependency here?

File: pom.xml [hunk 3]
```diff
@@ -36,11 +38,70 @@
...
+            <version>1.2.1</version>
+        </dependency>
+
+        <dependency>
+            <groupId>taglibs</groupId>
```
Comment (Sofasmile) [inline pom.xml]: the same?

File: pom.xml (hunk 2 above)
Comment (Rudiachenko) [reply pom.xml]: copied from prev homework :(

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**